internal/database/database.go

```
@@ -74,7 +73,7 @@ func SetupTestDB() (*sql.DB, func()) {
 		defer cancel()
 
 		// clean up
-		db2.ExecContext(ctx, fmt.Sprintf("DROP DATABASE "+util.Backquote(dbName)))
+		db2.ExecContext(ctx, "DROP DATABASE "+util.Backquote(dbName))
```

golangci-lint (1.64.8)

Error return value of `db2.ExecContext` is not checked

(errcheck)